p2p/lightchain.py

```diff
@@ -253,12 +253,18 @@ def peer_block_height(peer: LESPeer) -> int:
 
     async def _validate_header(self, header):
         if header.is_genesis:
-            parent_header = None
+            genesis_header = await self.headerdb.coro_get_canonical_block_header_by_number(0)
```

Comment:
As I said in https://github.com/ethereum/py-evm/issues/892#issuecomment-396387941, I think we should never download the genesis header as we must already have it on our first run, and we'll be comparing its hash with that of every peer we connect to. I think what we need to do is fix `LightPeerChain.get_sync_start_block()` to never request the genesis header